src/manager.rs

```diff
@@ -68,10 +68,15 @@ impl<T: MutEventSubscriber> SubscriberOps for EventManager<T> {
 impl<S: MutEventSubscriber> EventManager<S> {
     /// Create a new `EventManger` object.
     pub fn new() -> Result<Self> {
+        Self::new_with_capacity(256)
```

Comment:
Maybe we can add a define for this, something like: `DEFAULT_READY_EVENTS_CAPACITY` (or a better name :laughing: ).

Reply:
done 👍